File: 은지/LV1/문자열 내 p와 y의 개수.py
def solution(s):
    cnt_p = cnt_y = 0
    for x in s:
        if x == 'p' or x == 'P':
            cnt_p += 1
        elif x == 'y' or x == 'Y':
            cnt_y += 1
    if cnt_p == cnt_y:
        answer = True
    else:
        answer = False
    return answer

File: 은지/LV1/문자열 내 p와 y의 개수.js
function solution(s){
  var cnt_p = 0
  var cnt_y = 0
  for (let x of s) {
    if (x === 'p' || x === 'P') {
      cnt_p += 1
    }
    else if (x == 'y' || x === 'Y') {
      cnt_y += 1
    }
  }
  if (cnt_p === cnt_y) {
    var answer = true
  }
  else {
    var answer = false
  }

  return answer;
}